fetcher/Readme.md

```diff
@@ -0,0 +1,28 @@
+# Fetcher
+
+## Requirements
+
+- Node 6+
+
+## Install
+
+    # create a PostgreSQL database
+    createdb sm-fetcher
+    # create tables
+    psql -d sm-fetcher -f ../db/init-schema.sql
+    # install dependencies
+    npm install
+    # start web server
+    npm run dev
+    
+## Usage
+
+Use `curl` or `postman` or similar for the following:
+
+1. [Generate a GUID](https://www.guidgen.com/)
+2. [Create a Twitter app](https://apps.twitter.com/) or use an exiting one
+3. `POST http://locahlost:8081/registration/register` with `{"applicationId":"<YOUR GUID>"}`
+4. `POST http://locahlost:8081/twitter/register` with `{"applicationId":"<YOUR GUID>", "twitterId":"<TWITTER CONSUMER KEY>", "twitterSecret":"<TWITTER CONSUMER SECRET>"}`
+5. ???
```

Comment:
Once everything is registered you can request the analyzed data. Once again you can see how this is done here: https://github.com/jannikko/social-monitor/blob/master/web/server/services/ai.js in the `requestTopics` function. This won't yield a result until the data is fetched and analyzed. This can take a lot of time... you can see the intervals the data is fetched here: https://github.com/jannikko/social-monitor/blob/master/ai/app.py#L72